lib/image.coffee

```diff
@@ -45,7 +45,10 @@ bind = ($item, item) ->
       item.source
     else
       item.url
-    dialog.open item.text, """<img style="width:100%" src="#{url}">"""
+    dialogTitle = (item.text||'').replace /\[\[([^\]]+)\]\]/gi, ''
```

Comment:
What is the goal here? Does this just keep some markup out of the dialog title?

Reply:
This removes any links included in the image caption so they don't appear in the dialog's title.